README.md

```diff
@@ -1,12 +1,20 @@
 # reveal.js 
 
-A framework for easily creating beautiful presentations using HTML. [Check out the live demo](http://revealjs.com/).
+A framework for easily creating beautiful web-based presentations. [Check out the live demo](http://revealjs.com/).
 
->__Note:__ This repository was forked from [hakimel/reveal.js](https://github.com/hakimel/reveal.js) on 3/11/2019 to continue active development and feature contributions to reveal.js. Upstream This fork [is published](https://npmjs.com/package/@h3/reveal.js) on npmjs under the `@h3` scope. See [CONTRIBUTING](CONTRIBUTING.md) for info on how to join [HackerHappyHour](https://github.com/HackerHappyHour) and contribute!
+>__Note:__ This repository was forked from [hakimel/reveal.js](https://github.com/hakimel/reveal.js) on 3/11/2019 
+to continue active development and feature contributions to reveal.js. Upstream of this fork 
+[is published](https://npmjs.com/package/@h3/reveal.js) on npmjs under the `@h3` scope. 
+See [CONTRIBUTING](CONTRIBUTING.md) for info on how to join [HackerHappyHour](https://github.com/HackerHappyHour) and contribute!
 
 ## @h3/reveal.js Roadmap:
-* Native Docker integration
-* ... 
+
+The goal of @h3/reveal moving forward is to make it easier to use for non-developers and
+developers alike by modernizing how reveal is distributed, deployed, extended, and integrated.
+
+Our most important goals are currently tracked in our [projects][] and [milestones][]. In particular the [v2 milestone][]
```

Comment:
Either need `[v2 roadmap]` ref link here or change the link definition at bottom to `[v2 milestone]`.
```suggestion
Our most important goals are currently tracked in our [projects][] and [milestones][]. In particular the [v2 milestone][v2 roadmap]
```